src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -506,20 +503,4 @@ export class CreateScenariosComponent implements OnInit {
   goBackToPlanning() {
     this.router.navigate(['plan', this.plan$.value?.id]);
   }
-
-  downloadCsv() {
```

Comment:
I've moved this method to `scenario-results` component, as this create-scenarios one is kind of massive